### PR TITLE
DBDAART-1851_OTH_4_fields_with_potential_casing_differences

### PR DESCRIPTION
**Pull request by FICU-tensileai**
## What is this?
This is the first set of code updates so the fields being queried match proper casing of what is expected. Python matches the SAS/RedShift way .
Check for fields in various areas and checking against the upper field list to wrap with upper(). Would love to refactor a lot of this for consistency sake but can't this stage. Otherwise, its pretty straight forward.

## How would you classify this change (feature, bug, maintenance, etc.)?
maintenance

## How did I test this (if code change)?
ran through the OT_Runner to verify the SQL output in the logs

## Is there accompanying documentation for this change?
No

## Issues Encountered
None

*Optional section to include any challenges encountered, options considered, and the approach chosen.*

## PR Checklist
- [x] Is the issue number and a short description in the subject line?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

_Credit to TMSIS PR template and https://embeddedartistry.com/blog/2017/8/4/a-github-pull-request-template-for-your-projects_

### CODE REVIEW

**Review by TUAS-forpeople — COMMENTED** [review]
Looks good to me, but I'll just comment so that a data engineer can approve.
I appreciate the whitespace cleanup, did you use a tool for that or do it by hand?

**Comment by FICU-tensileai** [comment]
> Looks good to me, but I'll just comment so that a data engineer can approve. I appreciate the whitespace cleanup, did you use a tool for that or do it by hand?

I'm using the flake8, and black plugins plus I use the vs code cleanup shortcut which feels much like vim! Ctrl-K, Ctrl-X

File: taf/TAF_Closure.py [hunk 1]
```diff
@@ -953,16 +960,13 @@ def sum_paid(
...
-        ):
+    ):
         """
         Sum tot_mdcd_pd_amt on the headers OR mdcd_pd_amt on the lines where the given column equals the given condition.
         """
-         
 
         if not outcol:
-            _outcol = condcol1
-        else:
```
Comment (RIE9-adhoc) [inline taf/TAF_Closure.py:964]: Why was this part removed? It seems like this changes functionality that wouldn't be related to this ticket.

Comment (FICU-tensileai) [reply taf/TAF_Closure.py:964]: Fair question! For some reason the code was dead. The _outcol is not being used anywhere in the code. Beyond this it should say:
```if not outcol:
            outcol = condcol1```

Now. I believe that should be checked-in as it's in my branch.

Comment (RIE9-adhoc) [reply taf/TAF_Closure.py:964]: I see. I looked around and all calls to `sum_paid` all have a non-empty string `outcol` so this should be ok. I'd think `_outcol` is a private variable, but it just seems to be used elsewhere as a way to have 2 local variables with similar names.

**Review by RIE9-adhoc — APPROVED** [review]
